moskito-backend/src/Controller/BookmarkController.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace App\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\Routing\Annotation\Route;
+use App\Repository\UserRepository;
+use App\Repository\CampsiteRepository;
+use App\Serializer\UserSerializer;
+use App\Serializer\CampsiteSerializer;
+use App\Service\AuthenticationService;
+use App\Service\BookmarkService;
+
+class BookmarkController extends AbstractController
+{
+    /**
+     * @Route("/bookmark", methods={"GET"})
+     */
+    public function index( Request $request, AuthenticationService $authentication, CampsiteSerializer $serializer): JsonResponse
+    {   
+
+        $user = $authentication->isValid($request);
+        if (!$user) {
+            return $this->json(['error' => 'Not authorized'], JsonResponse::HTTP_UNAUTHORIZED);
+        }
+
+        $campsites = $user->getCampsite();
+        $campsitesAsArray = [];
+        foreach($campsites as $campsite) {
```

Comment:
würde die Variable sparen und hier direkt schreiben 
foreach($user->getCampsite() as ..:
müsste es nicht getCampsites() heißen?

Reply:
Warum die ArrayCollection in ein Array umwandeln? Wegen Serializer? 
Gibt es nicht auch eine toArray() methode?

Reply:
Absolut! Danke!